src/main/java/software/amazon/cloudwatchlogs/emf/logger/MetricsLogger.java

```diff
@@ -257,10 +265,112 @@ public MetricsLogger putMetric(String key, double value, Unit unit)
      * @throws InvalidMetricException if the metric is invalid
      */
     public MetricsLogger putMetric(String key, double value) throws InvalidMetricException {
-        this.putMetric(key, value, Unit.NONE, StorageResolution.STANDARD);
+        this.putMetric(
+                key, value, Unit.NONE, StorageResolution.STANDARD, this.defaultAggregationType);
+        return this;
+    }
+
+    /**
+     * Put a metric value. This value will be emitted to CloudWatch Metrics asynchronously and does
+     * not contribute to your account TPS limits. The value will also be available in your
+     * CloudWatch Logs
```

Comment:
nit - not a blocker: Personal flavour, but for overloaded method java doc, I prefer to use @see or something if the documentation is identical. Makes it easier to update in the future.